### ISSUE TITLE
Episode detected as movie because year is in the title

### ISSUE THREAD

**Issue by dvkch** [guessit-io/guessit#428]
Hello,

I found that some tv show episodes are detected as a movie, I think because the title contains a year. Here is an example :

```
For: the.flash.2014.208.hdtv-lol[ettv].mkv
GuessIt found: {
    "title": "the flash", 
    "year": 2014, 
    "episode_title": "208", 
    "format": "HDTV", 
    "release_group": "lol[ettv]", 
    "container": "mkv", 
    "mimetype": "video/x-matroska", 
    "type": "movie"
}
```

The presence of an `episode_title` could be an indicator of a tv show and help prevent this incorrect match.

**Comment by ratoaq2** [comment]
You could use `--type episode`:

```
--type episode the.flash.2014.208.hdtv-lol[ettv].mkv
For: the.flash.2014.208.hdtv-lol[ettv].mkv
GuessIt found: {
    "title": "the flash", 
    "year": 2014, 
    "season": 2, 
    "episode": 8, 
    "format": "HDTV", 
    "release_group": "lol[ettv]", 
    "container": "mkv", 
    "mimetype": "video/x-matroska", 
    "type": "episode"
}
```

**Comment by dvkch** [comment]
Thanks that's a nice workaround! Unfortunately I use guessit to automatically organize a folder containing episodes and movies, I guess if I find a episode number I can try to restart guessit with a forced type and act accordingly.

**Comment by ratoaq2** [comment]
This is the same issue as this one: #235

**Comment by ratoaq2** [comment]
Fixed by #446
